components/brave_ads/browser/background_helper.cc

```diff
@@ -35,7 +35,7 @@ void BackgroundHelper::TriggerOnForeground() {
   }
 }
 
-#if !defined(OS_MACOSX) && !defined(OS_WIN) && !defined(OS_LINUX) && !defined(OS_ANDROID)  // NOLINT
+#if !defined(OS_APPLE) && !defined(OS_WIN) && !defined(OS_LINUX) && !defined(OS_ANDROID)  // NOLINT
```

Comment:
Should be `OS_MAC`

Reply:
Addressed by 0230f8e014a698a5cf5abd780734aa5bae433b3f